### PR TITLE
Minimum Viable Product

### PR DESCRIPTION
**Pull request by meronogbai**
- App lets you get the weather data of any city in the world
- Responsive design using bootstrap.

### CODE REVIEW

File: src/weather-api.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+async function getWeatherData(location, apiKey) {
```
Comment (mosaaleb) [inline src/weather-api.js]: - [x] It uses ES6 syntax. Kindly let's turn this function syntax to arrow function.

File: src/weather-api.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  const endpoint = `https://api.openweathermap.org/data/2.5/weather?q=${location}&APPID=${apiKey}`;
+  const data = await fetch(endpoint);
+  const weatherData = await data.json();
+  return weatherData;
+}
+
+export default async function formattedWeatherData(location, apiKey) {
```
Comment (mosaaleb) [inline src/weather-api.js]: - [x] Same here. Arrow function syntax.

File: src/display.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+function celsiustoFahrenheit(degree) {
```
Comment (mosaaleb) [inline src/display.js]: - [x] Let's make sure to convert all the function to arrow function syntax in this file.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Changes required

Well done @meronokbay :clap:. You have handled most of the requirements. Nice job adding the loading feature as well. Yet, there are other requirements we need to work on before moving on. Kindly take a look at my comments below.


- [x] It changes its appearance based on the received data


- [ ] **[OPTIONAL]** It fetches the current geolocation of the user when the app is first accessed and fetches the weather from that location.

Please handle these requirements and submit a new code review.

##### Muhammad Ebeid

**Review by bellom — APPROVED** [review]
## Hi @meronokbay  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**